app/src/main/java/chat/rocket/android/renderer/MessageRenderer.java

```diff
@@ -76,10 +75,11 @@ public MessageRenderer timestampInto(TextView textView) {
     }
 
     switch (object.getSyncState()) {
```

Comment:
@filipedelimabrito As I understand so far, NOT_SYNCED is the first state of a message before it's sent, when it's first created.